src/client/components/DemographicsTooltip.tsx

```diff
@@ -67,10 +72,20 @@ const DemographicsTooltip = ({
     (population: number) =>
       (demographics.population ? population / demographics.population : 0) * 100
   );
-  const races = ["white", "black", "asian", "hispanic", "other"] as const;
-  const rows = races.map((id: typeof races[number]) => (
-    <Row key={id} label={id} percent={percentages[id]} color={demographicsColors[id]} />
-  ));
+  const races = [
+    "white",
+    "black",
+    "asian",
+    "hispanic",
+    "nativeAmerican",
+    "pacificIslander",
+    "other"
+  ] as const;
+  const rows = races
+    .filter(race => percentages[race])
```

Comment:
We want to continue to show the row if the field is there but the value is `0`
```suggestion
    .filter(race => percentages[race] !== undefined)
```